scripts/publish

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+tagPushed=$(git ls-remote --tags origin | grep -c "$npm_package_version")
+
+if [ "$tagPushed" -eq "0" ]; then echo "Current version tag doesn't exist on remote." && exit 1; fi
+
+headerLines=$(grep -n "##" CHANGELOG.md | cut -f1 -d:)
+lineNos=()
+while IFS='' read -r line; do lineNos+=("$line"); done < <(echo "$headerLines")
+
+releaseNotes=$(sed -n -e "${lineNos[1]}",$((lineNos[2]-1))p CHANGELOG.md)
+
+hub release create -m "$releaseNotes" "$npm_package_version"
```

Comment:
I actually don't know if this will work, or if it needs to be 

```suggestion
hub release create -m "$releaseNotes" "v$npm_package_version"
```

Reply:
Just tested it, needs the `v`. gonna make an additional little edit though bc we don't need the "##" in the title.